src/main/resources/application.yaml

```diff
@@ -99,4 +99,6 @@ feign:
 resendInterval: 60
 
 # sync time interval in minutes
-syncInterval: 60
\ No newline at end of file
+syncInterval: 60
+
+logging-component-name: "RD_User_Profile_API"
```

Comment:
remove ""

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-user-profile-api/pull/395/commits/037909c45ca61e983546303bf3e8729e773401bc